.github/workflows/docker.yml

```diff
@@ -1,28 +0,0 @@
-name: docker
```

Comment:
We do not need this image since it will now be tested as part of each of the app's workflows.